.travis.yml

```diff
@@ -19,51 +19,57 @@ env:
     # configuration.
     ###########################################################################
 
-    # Check the matrix of:
+    # Check a subset of the matrix of:
     #   LLVM  : {2.9, 3.4}
     #   SOLVERS : {Z3, STP, STP:Z3, metaSMT}
     #   STP_VERSION   : {2.1.2, master}
     #   METASMT_DEFAULT : {btor, stp, z3}
     #   UCLIBC: {0, klee_uclibc_v1.0.0, klee_0_9_29} // Note ``0`` means disabled
-    # with Asserts enabled.
+    #   DISABLE_ASSERTIONS: {0, 1}
+    #   ENABLE_OPTIMIZED: {0, 1}
+    #   COVERAGE: {0, 1}
+    #   USE_TCMALLOC: {0, 1}
+    #   USE_CMAKE: {0, 1}
 
     # COVERAGE set indicated that coverage data should be uplaoded to the server, only ONE job should have COVERAGE set
```

Comment:
COVERAGE is broken. The server receiving that information is dead. Also we might have been leaking credentials (see our internal slack about this).